src/GCPClientManager.php

```diff
@@ -14,22 +15,25 @@
 use Google\Task\Runner;
 use Google_Client;
 use Google_Service_CloudResourceManager;
+use GuzzleHttp\Client;
 use GuzzleHttp\Client as GuzzleClient;
+use GuzzleHttp\HandlerStack;
 use Keboola\StorageDriver\BigQuery\Client\BigQuery\Retry;
 use Keboola\StorageDriver\Credentials\GenericBackendCredentials;
 use Keboola\TableBackendUtils\Connection\Bigquery\BigQueryClientWrapper;
 use Psr\Log\LoggerInterface;
+use Throwable;
 
 class GCPClientManager
 {
     public const CONNECTION_TIMEOUTS_MAP = [
         'timeout' => self::TIMEOUT,
         'connect_timeout' => self::CONNECT_TIMEOUT,
     ];
-
+    private const KEBOOLA_USER_AGENT = 'Keboola/1.0 (GPN:Keboola; connection)';
```

Comment:
Skus prosim niekde zistit predtim ako to budeme pouzivat akeho user agenta tam teda chceme pouzivat, reps. Google by asi mal vedet teda co tam posielame

Reply:
To je 1:1 stím co tam posílá transformace, takže tento

Reply:
https://github.com/keboola/google-bigquery-transformation/blob/main/src/BigQueryConnection.php#L66